### ISSUE TITLE
About calculate noise rate by using clean eval-dataset

### ISSUE THREAD

**Issue by Yuxin-Du-Lab** [nazmul-karim170/UNICON-Noisy-Label#9]
Thank you very much for your interesting work. 
I have a question about [this line](https://github.com/nazmul-karim170/UNICON-Noisy-Label/blob/647b3ee9f4e71faf1794cf47a974c9772d43dfbe/Train_cifar.py#L291): in noise learning, is it a normal way to use a clean eval-set to calculate the noise rate (or something similar)? (If you have a test set to do the final test)
I'm just new to this field and don't know the rules very well.
Looking forward to your answer.

**Comment by Yuxin-Du-Lab** [comment]
Add a little bit of my understanding: assuming a manually set ratio, the trainer can adjust the hyperparameter using the result of eval. Your work just automates the process.
Is this the correct understanding?

**Comment by nazmul-karim170** [comment]
To your first question, the noise rate is calculated from the training set.  We use a clean eval-set for calculating the validation accuracy. 

To your second question, yes, we do not adjust the hyperparameters for different noise rates. Same HP for all noise rates.
